hybridse/src/udf/default_udf_library.cc

```diff
@@ -666,9 +666,37 @@ void DefaultUdfLibrary::InitStringUdf() {
                 select date_format(date(1590115420000),"%Y-%m-%d");
                 --output "2020-05-22"
             @endcode)");
+    RegisterExternal("upper")
+        .args<StringRef, StringRef>(
+            static_cast<void (*)(codec::StringRef*, codec::StringRef*, bool*)>(udf::v1::upper))
+        .return_by_arg(true)
+        .doc(R"(
```

Comment:
https://github.com/4paradigm/OpenMLDB/pull/782/files#diff-ec3be29d000ba60c534a41720af199bb704674e88508d9b6224c8359b8d078bb
> Given the result is a nullable date type, we configure  **return_by_arg** as ***true*** and return type as `Nullable<Date>`

Maybe I should emphasize the usage in the ExternalFuncRegistryHelper API's description.
- RegisterExternal(function_name): ....
- ...
- returns<return_type>: configure return type. It is worth nothing that when the return type is **nullable**, we should configure it as `returns<Nullable<type>>` explicitly.